File: Source/ses_io/src/wrappers/fortran/wrap_ses_exit.c

#include "ses_defines.h"
#include "ses_globals.h"
#include "ses_internals.h"
#include "ses_externs.h"

#include "stdio.h"
#include <stdlib.h>

#include <stdio.h>

#undef DEBUG_WRAP

#ifdef LC_UNDER
ses_boolean ses_exit_(void) {
#else

#ifdef UC_UNDER
ses_boolean SES_EXIT_(void) {
#else

#ifdef LC_NOUNDER
ses_boolean ses_exit(void) {
#else

#ifdef UC_NOUNDER
ses_bolean SES_EXIT(void) {
#endif

#endif
#endif
#endif

  ses_boolean return_value = SES_FALSE;

  return_value = ses_exit();

  return return_value;

}

File: Source/ses_io/src/wrappers/fortran/wrap_ses_exit.f90

#undef DEBUG_WRAP

SES_BOOLEAN function ses_exit_f90()

   implicit none

#define DEBUG_32BIT
#ifdef DEBUG_32BIT
   integer(kind = 4) :: return_4
   integer(kind = 8) :: return_8
#endif

   SES_BOOLEAN :: ses_exit, return_value

   return_value = ses_exit()

   ses_exit_f90 = return_value

end function ses_exit_f90


